### PR TITLE
Add Silversun flower pin

### PR DESCRIPTION
**Pull request by TheGoret**
Lore request by Schwann.

Thanks to Happy_Fox and VT for the sprites.

### CODE REVIEW

**Comment by TheGoret** [comment]
!review

**Comment by Chada-1** [comment]
I'm not going to suggest changes because _I don't know how_ 👀 But you put an 'after after' in the description which is a light typo

**Comment by NotSchwann** [comment]
> I'm not going to suggest changes because _I don't know how_ 👀 But you put an 'after after' in the description which is a light typo

Uh yeah my intent with that typo to make sure everybody read the description haha _haha_ I am quite smart as you can see.

File: html/changelogs/goret-silversun-flower.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Added a silversun flower pin, related to the Silversun lore. Thanks to Happy_Fox and VT for the sprites."
```
Comment (Arrow768) [inline html/changelogs/goret-silversun-flower.yml]: Additional contributors should be added to the author and not the change message.

**Pull request merged**